File: go/1150.go
package main

import "fmt"

func main() {
  var x, z, count, sum int
  fmt.Scanf("%d", &x)

  for {
    fmt.Scanf("%d", &z)

    if z > x {
      break
    }
  }

  sum = x
  count = 1

  for {
    sum += x + count
    count += 1
    if sum > z {
      break
    }
  }

  fmt.Println(count)
}

File: lua/1150.lua
local x, z, count, sum

x = io.read("*n")

repeat
    z = io.read("*n")
until z > x

sum = x
count = 1

while true do
    sum = sum + x + count
    count = count + 1

    if sum > z then
        break
    end
end

print(count)
